internal/runners/packages/list.go

```diff
@@ -154,12 +189,23 @@ type packageRow struct {
 	Version string `json:"version" locale:"package_version,Version"`
 }
 
-func newFilteredRequirementsTable(requirements []*gqlModel.Requirement, filter string, nstype model.NamespaceType) []packageRow {
+func newFilteredRequirementsTable(requirements []*gqlModel.Requirement, filter string, nstype model.NamespaceType, rt *runtime.Runtime, ns *model.Namespace) []packageRow {
```

Comment:
Nit: Passing in the resolved artifacts instead feels cleaner.

Reply:
Good call.